.env.example

```diff
@@ -0,0 +1,2 @@
+AZURE_PROCESSOR_KEY=
+AZURE_PROCESSOR_ENDPOINT=https://pipeline-1-text-extraction.cognitiveservices.azure.com/
```

Comment:
I would say we don't want the processor endpoint in the code base anywhere as this is our personal instantiated processor not a generic azure endpoint. 

I would say we set as a repository secret and pass into the github actions work flow as is done here: 
`https://github.com/climatepolicyradar/navigator-data-pipeline/blob/main/.github/workflows/test.yml#L61`